LCM/dsc/engine/mof/MSFT_DSCMetaConfiguration.mof

```diff
@@ -92,4 +92,9 @@ class MSFT_DSCMetaConfiguration
   
    [Read, Description("AgentId of the current Dsc Agent.")]
    string AgentId;   
+   
+   
+   boolean EnableSignatureValidation;
```

Comment:
I don't understand why we have two contradicting flags. What happens if they are both set to true?
Is there a subtle difference between Enable(?)Signature and Disable**Module**Signature?
I suggest having a single `EnableModuleSignatureValidation` flag. When set to false, it means signature validation is disabled.